_data/people.yml

```
@@ -11,6 +11,10 @@
     url: "bn"
     role: "Software Developer"
     image: bn.jpg
+  - name: "Enrique Miralles-Dolz"
+    url: "emd"
+    role: "Associate Research Engineer, PPPL"
```

I wonder if it makes sense to define you as a visitor, e.g. 
```suggestion
    role: "Associate Research Engineer, (visiting from) PPPL "
```